src/lib.rs

```diff
@@ -173,7 +173,7 @@ const ORIGINAL_MAX: u64 = 0;
 /// of percentiles.
 pub trait Counter
     : num::Num + num::ToPrimitive + num::FromPrimitive + num::Saturating + num::CheckedSub
-    + num::CheckedAdd + Copy + PartialOrd<Self>{
+    + num::CheckedAdd + Copy + PartialOrd<Self> {
```

Comment:
Yeah, that's not a bad idea... On other projects I've set up git hooks, etc; maybe I'll port one of those over to make it easy for others to adopt. Consistency is good.